### DISCUSSION TITLE
Are 1.7 charts compatible with 1.8?

### DISCUSSION THREAD

**Discussion by AnnoyingAlchemist** [KadeArchive/Kade-Engine#2817] [Q&A]
Tried to move some charts from 1.7 to 1.8, but it seems like it doesn't work. Do I have to rechart my songs in 1.8 or is there a way to make 1.7 charts work with 1.8?

**Comment by RealDealJinx** [comment]
all versions are

**Reply by glowsoony** [reply]
not quite true as if you try to import 1.8 charts into any version that is older like 1.6.2-1.5 the 1.7-1.8 charts will not work cause of the new code in the playstate.hx in 1.7-1.8. and cause of the older versions lacking the LuaClass.hx.

**Reply by RealDealJinx** [reply]
no i mean all versions like you can port a 1.6.2 chart to 1.8 but you cant port a 1.8 chart to 1.6.2

**Reply by perngjiafu** [reply]
dk. mine from a psych engine port, all of the 1000 notes i placed are cramped in one place

**Comment by BoloVEVO** [comment]
Some sections in older KE versions, or in other engine are pretty messed up if you port it to recent KE versions.
